node/service/src/lib.rs

```diff
@@ -711,6 +717,13 @@ where
 
 		Some(backoff)
 	};
+	// For now BEEFY is explicitly disabled on non-test chains.
+	if config.chain_spec.is_polkadot() ||
+		config.chain_spec.is_kusama() ||
+		config.chain_spec.is_westend()
+	{
+		enable_beefy = false;
```

Comment:
I would add a `warn`: `if enable_beefy { warn!("Tried to enable BEEFY on a production network. Refusing as BEEFY is still experimental.") }`. Something along those lines.

Reply:
I thought about that as well, but I found it odd that `log` crate is not used anywhere in `node/service` and I just assumed there's a reason for it...

I'll add `log` dependency and log a `warn!`, but writing it out here as well so anybody who knows of a reason we shouldn't, can react :smile:

Reply:
There is a local crate for logging that also deals with tracing `gum::warn!` should work.

Reply:
I should document this more broadly, but here's the rationale for the interested reader https://github.com/paritytech/polkadot/blob/6cafab2d926000fa612af29b02e8ed482d4dd9c5/node/gum/README.md